### DISCUSSION TITLE
Getting TypeError in CLU package code

### DISCUSSION THREAD

**Discussion by pmin20** [google/flax#3411] [Q&A]
File "/Users/paigemin/opt/miniconda3/envs/thesis/lib/python3.9/site-packages/clu/parameter_overview.py", line 28, in <module>
    _ParamsContainer = dict[str, np.ndarray] | Mapping[str, Mapping[str, Any]]
TypeError: unsupported operand type(s) for |: 'types.GenericAlias' and 'types.GenericAlias'

I'm getting the above error from the CLU package upon installing, and am not sure how to resolve it. Please help!

**Comment by chiamp** [comment]
What version of Python are you using and what packages did you already have installed before trying to install `clu` (what output do you get when you type `pip freeze`)? Were you trying to install `clu` directly, or it was attempting to install it as a dependency of another package?

**Reply by pmin20** [reply]
Hi! I'm using python 9.8, and I believe at the time that I install clu (onto my conda environment) I had packages such as jax, jaxlib, flax, tensorflow, and scenic installed. I've attached the output I get from pip freeze. I installed clu with "pip install clu" because it is a package used in the open source code I am working with. I'm newer to virtual environments and managing numerous packages so any help would be much appreciated!

[pipfreeze.txt](https://github.com/google/flax/files/12888426/pipfreeze.txt)

**Reply by chiamp** [reply]
Sorry for the delay. Could you upgrade to the latest version of Flax and try again?
